DayrSuryan/DSEthiop19.xml

```diff
@@ -289,7 +289,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                              The scribe executed the word divider <foreign xml:lang="gez">፡</foreign>
                              in a peculiar way. He interpreted as dot
                              the last letter element protruding to the right,
-                             (cp., such letters as ኒ, ዱ, ተ, ኪ, but also ም), and
+                             (cp., e.g., such letters as ኒ, ዱ, ተ, ኪ, but also ም), and
```

Comment:
Is this Oxford style? I find them a lot of abbreviations and commas..I was told that no comme should be used after e.g.

Reply:
@SusanneHummel can you help us one last time in this regard :) then we will hopefully learn

Reply:
there should be no comma after e.g.
I don't have the book here. i think my copy is still with @DenisNosnitsin1970 
I once made a photo of this entry from the manual for a BodCat Ms record. Unfortunately, I don't remember which one...
Why do you need cp?

Reply:
Thank you!

Reply:
I see in the Oxford Style book that no comma follows e.g. But now I see that "e.g." is not necessary here. I am deleting it.

Reply:
Cp. for "compare"?

Reply:
I found both cp. and e.g. superfluous. As you also noticed.